src/presentation/http/router/note.ts

```diff
@@ -19,10 +20,10 @@ const NoteRouter: FastifyPluginCallback = (fastify, _, done) => {
    * Add new note
    */
   fastify.post('/add', async (request, reply) => {
-    reply.send(request.query);
+    reply.send(NoteService.addNote(request.query));
```

Comment:
don't pass query directly to service. Extract needed params and pass them explicitly

Reply:
done